natss/pkg/reconciler/dispatcher/natsschannel.go

```diff
@@ -59,7 +58,7 @@ const (
 type Reconciler struct {
 	*reconciler.Base
 
-	natssDispatcher *dispatcher.SubscriptionsSupervisor
+	natssDispatcher dispatcher.NatssDispatcher
 
 	natsschannelLister   listers.NatssChannelLister
 	natsschannelInformer cache.SharedIndexInformer
```

Comment:
Doesn't look like `natsschannelInformer` is used anywhere. Dead code?

Reply:
True. I tried to not change too much code but only adding tests. But since I already didn't follow my own idea, let's change this one too :-D Good catch!